addons/website/static/tests/builder/images.test.js

```diff
@@ -36,8 +38,20 @@ test("double click on Image", async () => {
     await dblclick(":iframe img.a_nice_img");
     await animationFrame();
     expect(".modal-content:contains(Select a media) .o_upload_media_button").toHaveCount(1);
+    expect("div.o-tooltip").toHaveCount(0);
 });
 
+test("simple click on Image", async () => {
+    await setupWebsiteBuilder(`<div><img class=a_nice_img src='${dummyBase64Img}'></div>`);
+    expect(".modal-content").toHaveCount(0);
+    await click(":iframe img.a_nice_img");
+    await waitFor("div.o-tooltip");
+    expect("div.o-tooltip").toHaveCount(1);
```

Comment:
It s strange to check if you don t have .modal-content and you don t check it at the end

Reply:
You can remove this check